src/main/java/com/cburch/logisim/std/arith/MultiplierHdlGeneratorFactory.java

```diff
@@ -33,68 +33,68 @@ public MultiplierHdlGeneratorFactory() {
         .add(CALC_BITS_STRING, CALC_BITS_ID, HdlParameters.MAP_MULTIPLY, 2)
         .add(UNSIGNED_STRING, UNSIGNED_ID, HdlParameters.MAP_ATTRIBUTE_OPTION, Comparator.MODE_ATTR, ComparatorHdlGeneratorFactory.SIGNED_MAP);
     myWires
-        .addWire("s_mult_result", CALC_BITS_ID)
-        .addWire("s_extended_Cin", CALC_BITS_ID)
-        .addWire("s_new_result", CALC_BITS_ID);
+        .addWire("s_multResult", CALC_BITS_ID)
+        .addWire("s_extendedcarryIn", CALC_BITS_ID)
+        .addWire("s_newResult", CALC_BITS_ID);
     myPorts
-        .add(Port.INPUT, "INP_A", NR_OF_BITS_ID, Multiplier.IN0)
-        .add(Port.INPUT, "INP_B", NR_OF_BITS_ID, Multiplier.IN1)
-        .add(Port.INPUT, "Cin", NR_OF_BITS_ID, Multiplier.C_IN)
-        .add(Port.OUTPUT, "Mult_lo", NR_OF_BITS_ID, Multiplier.OUT)
-        .add(Port.OUTPUT, "Mult_hi", NR_OF_BITS_ID, Multiplier.C_OUT);
+        .add(Port.INPUT, "inputA", NR_OF_BITS_ID, Multiplier.IN0)
+        .add(Port.INPUT, "inputB", NR_OF_BITS_ID, Multiplier.IN1)
+        .add(Port.INPUT, "carryIn", NR_OF_BITS_ID, Multiplier.C_IN)
+        .add(Port.OUTPUT, "multLow", NR_OF_BITS_ID, Multiplier.OUT)
+        .add(Port.OUTPUT, "multHigh", NR_OF_BITS_ID, Multiplier.C_OUT);
   }
 
   @Override
   public LineBuffer getModuleFunctionality(Netlist TheNetlist, AttributeSet attrs) {
-    final var contents =
-          LineBuffer.getBuffer()
-            .pair("nrOfBits", NR_OF_BITS_STRING)
-            .pair("unsigned", UNSIGNED_STRING)
-            .pair("calcBits", CALC_BITS_STRING);
-
+    final var contents = LineBuffer.getHdlBuffer()
+        .pair("nrOfBits", NR_OF_BITS_STRING)
+        .pair("unsigned", UNSIGNED_STRING)
+        .pair("calcBits", CALC_BITS_STRING);
     if (Hdl.isVhdl()) {
-      contents.add("""
-          s_mult_result <= std_logic_vector(unsigned(INP_A)*unsigned(INP_B))
-                              WHEN {{unsigned}}= 1 ELSE
-                           std_logic_vector(signed(INP_A)*signed(INP_B));
-          s_extended_Cin({{calcBits}}-1 DOWNTO {{nrOfBits}}) <= (OTHERS => '0') WHEN {{unsigned}} = 1 ELSE (OTHERS => Cin({{nrOfBits}}-1));
-          s_extended_Cin({{nrOfBits}}-1 DOWNTO 0) <= Cin;
-          s_new_result  <= std_logic_vector(unsigned(s_mult_result) + unsigned(s_extended_Cin))
-                              WHEN {{unsigned}}= 1 ELSE
-                           std_logic_vector(signed(s_mult_result) + signed(s_extended_Cin));
-          Mult_hi       <= s_new_result({{calcBits}}-1 DOWNTO {{nrOfBits}});
-          Mult_lo       <= s_new_result({{nrOfBits}}-1 DOWNTO 0);
+      contents.empty().addVhdlKeywords().add("""
+          s_multResult <= std_logic_vector(unsigned(inputA)*unsigned(inputB))
+                              {{when}} {{unsigned}}= 1 {{else}}
```

Comment:
wouldn't it be more readable to move {{else}} to next line and align with {{when}}? (multiple)